continuous_integration/azure/install.bat

```diff
@@ -34,7 +34,7 @@ if %USE_CONDA% == yes (
 )
 
 :: Install spyder-kernels from master
-pip install -q --no-deps git+https://github.com/spyder-ide/spyder-kernels
+pip install -q --no-deps git+https://github.com/spyder-ide/spyder-kernels.git@refs/pull/182/head
```

Comment:
@impact27, please revert the changes to our installation scripts because I just merged PR spyder-ide/spyder-kernels#182.